xpti/CMakeLists.txt

```diff
@@ -1,17 +1,28 @@
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.8)
 
 set(XPTI_VERSION 0.4.1)
+project (xpti VERSION "${XPTI_VERSION}" LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+
 set(XPTI_DIR ${CMAKE_CURRENT_LIST_DIR})
 # Setting the same version as SYCL
 set(CMAKE_CXX_STANDARD 11)
 
+option(XPTI_ENABLE_WERROR OFF)
```

Comment:
Why not have this default to ON here?

Reply:
New compilers introduce new warnings, which is inconvenient for users. So, we have this enabled in CI (where we have control over compiler versions), but disabled by default for those who intend to just use this project (rather than contribute).